pinc/ProjectState.inc

```diff
@@ -69,7 +72,11 @@ class ProjectStates
 // Note that the order in which these project states are declared
 // is the order in which they will be displayed in various contexts
 // (via ProjectStates::get_states()).
-function declare_project_states($rounds, $pools)
+/**
```

Comment:
Let's add a blank line between the comment blocks to make it clearer they are separate.
```suggestion

/**
```